### DISCUSSION TITLE
config/retronas_packages.ini accessible in SMB share

### DISCUSSION THREAD

**Discussion by bwann** [retronas/discussions#125] [Q&A] [answered]
Hi, first time Retronas user. I installed the samba server and noticed when I browsed to the new share \\RETROSMB\retronas, the "config" directory and contents like retronas_packages.ini and modules directory were accessible and could be modified. This happened even if I changed the Data Directory from the config TUI from say /data/retronas to /data/shared.

I didn't notice that in the YouTube setup videos, and looking at the ansible playbooks they all use "retronas_path" for the path in samba config and retronas_system_config. Is exposing these config files within the share intentional? Will it mess up Retronas if these files/directories are modified or deleted? It looks like they'll just be written again the next time ansible is applied. Seems like this will be the case for other service such as Netatalk as well.

**Comment by sairuk** [comment, marked as answer]
The videos are over 2y old and the core of the project has moved on, you are correct in that the folders you mentioned will be recreated as part of setup/install of components. 

They are exposed over the share so the are available for backups, restoration and installation of custom local modules as per docs

If you change the data directory after installing services all previously installed services will need to be reinstalled as the retronas_path variable is populated by ansible during installation.

**Reply by bwann** [reply]
Ok, thanks for the info!
